### PR TITLE
[DSD-3932] updated push values.yml and Chart.yaml file for upgrade-B4 release

### PR DESCRIPTION
**Pull request by Prafulrakhade**

### CODE REVIEW

File: charts/admin-ui/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: admin-ui
 description: A Helm chart for MOSIP Admin UI module
 type: application
-version: 12.0.1
+version: 12.0.1-B3
```
Comment (ckm007) [inline charts/admin-ui/Chart.yaml:5]: 1.2.0.1-B1

File: charts/authmanager/values.yaml [hunk 2]
```diff
@@ -51,8 +51,8 @@ service:
 
 image:
   registry: docker.io
-  repository: mosipqa/kernel-auth-service
-  tag: 1.2.0.1
+  repository: mosipid/kernel-auth-service
+  tag: 1.2.0.1-b3
```
Comment (ckm007) [inline charts/authmanager/values.yaml:55]: 1.2.0.1-B3

File: charts/keymanager/values.yaml [hunk 3]
```diff
@@ -52,7 +52,7 @@ service:
 
 image:
   registry: docker.io
-  repository: mosipqa/kernel-keymanager-service
+  repository: mosipid/kernel-keymanager-service
   tag: 1.2.0.1
```
Comment (ckm007) [inline charts/keymanager/values.yaml:56]: 1.2.0.1-B3

File: charts/minio-client-util/Chart.yaml [hunk 4]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: minio-client-util
 description: A Helm chart to deploy s3 utility to remove apitestrig reports from minio
 type: application
-version: 1.0.0-P1
+version: 1.0.0
```
Comment (ckm007) [inline charts/minio-client-util/Chart.yaml:5]: 1.0.0-P2

File: charts/mock-smtp/values.yaml [hunk 5]
```diff
@@ -77,8 +77,8 @@ services:
 
 image:
   registry: docker.io
-  repository: mosipqa/mock-smtp
-  tag: 0.0.2
+  repository: mosipid/mock-smtp
+  tag: 0.0.1
```
Comment (ckm007) [inline charts/mock-smtp/values.yaml:81]: 0.0.2

File: charts/pmp-ui/Chart.yaml [hunk 6]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: pmp-ui
 description: A Helm chart for MOSIP pmp UI module
 type: application
-version: 12.0.1
+version: 12.0.1-B3
```
Comment (ckm007) [inline charts/pmp-ui/Chart.yaml:5]: 1.2.0.1-B2

File: charts/prereg-booking/values.yaml [hunk 7]
```diff
@@ -52,8 +52,8 @@ service:
 
 image:
   registry: docker.io
-  repository: mosipqa/pre-registration-booking-service
-  tag: 1.2.0.1
+  repository: mosipid/pre-registration-booking-service
+  tag: 1.2.0.1-B3
```
Comment (ckm007) [inline charts/prereg-booking/values.yaml:56]: 1.2.0.1-B2

File: charts/prereg-captcha/values.yaml [hunk 8]
```diff
@@ -52,8 +52,8 @@ service:
 
 image:
   registry: docker.io
-  repository: mosipqa/pre-registration-captcha-service
-  tag: 1.2.0.1
+  repository: mosipid/pre-registration-captcha-service
+  tag: 1.2.0.1-B3
```
Comment (ckm007) [inline charts/prereg-captcha/values.yaml:56]: 1.2.0.1-B2

File: charts/prereg-datasync/values.yaml [hunk 9]
```diff
@@ -52,8 +52,8 @@ service:
 
 image:
   registry: docker.io
-  repository: mosipqa/pre-registration-datasync-service
-  tag: 1.2.0.1
+  repository: mosipid/pre-registration-datasync-service
+  tag: 1.2.0.1-B3
```
Comment (ckm007) [inline charts/prereg-datasync/values.yaml:56]: 1.2.0.1-B2

File: charts/prereg-gateway/Chart.yaml [hunk 10]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: prereg-gateway
 description: A Helm chart for MOSIP Pre-registration Istio Gateway
 type: application
-version: 12.0.1-B2
+version: 12.0.1-B3
```
Comment (ckm007) [inline charts/prereg-gateway/Chart.yaml:5]: 12.0.1-B2

File: charts/prereg-ui/Chart.yaml [hunk 11]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: prereg-ui
 description: A Helm chart for MOSIP Pre-registration UI
 type: application
-version: 12.0.1
+version: 12.0.1-B2
```
Comment (ckm007) [inline charts/prereg-ui/Chart.yaml:5]: 12.0.1-B1

File: charts/regclient/values.yaml [hunk 12]
```diff
@@ -51,8 +51,8 @@ service:
 
 image:
   registry: docker.io
-  repository: mosipqa/registration-client
-  tag: 1.2.0.1
+  repository: mosipid/registration-client
+  tag: 1.2.0.1-B2
```
Comment (ckm007) [inline charts/regclient/values.yaml:55]: update regclient.version

File: charts/regproc-opencrvs/Chart.yaml [hunk 13]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: regproc-opencrvs
 description: A Helm chart for mosip regproc-opencrvs stage.
 type: application
-version: 12.0.1
+version: 12.0.1-B4
```
Comment (ckm007) [inline charts/regproc-opencrvs/Chart.yaml:5]: 12.0.1-B3

File: charts/websub/values.yaml [hunk 14]
```diff
@@ -52,8 +52,8 @@ service:
 
 image:
   registry: docker.io
-  repository: mosipqa/websub-service
-  tag: 1.2.0.1
+  repository: mosipid/websub-service
+  tag: 1.2.0.1-B3
```
Comment (ckm007) [inline charts/websub/values.yaml:56]: 1.2.0.1-B2

File: charts/mock-smtp/values.yaml (hunk 5 above)
Comment (Prafulrakhade) [reply charts/mock-smtp/values.yaml:81]: mock smtp not released sir

File: charts/minio-client-util/Chart.yaml (hunk 4 above)
Comment (Prafulrakhade) [reply charts/minio-client-util/Chart.yaml]: this is not released so pointing to last released chart version

File: charts/mock-smtp/values.yaml (hunk 5 above)
Comment (ckm007) [reply charts/mock-smtp/values.yaml:81]: release mock smtp 0.0.2 and update relevant changes

**Review by ckm007 — APPROVED** [review]

**Pull request merged**